.github/workflows/0-start.yml

```
@@ -36,7 +36,7 @@ jobs:
     steps:
       # We'll need to check out the repository so that we can edit the README
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.3.0
```

```suggestion
        uses: actions/checkout@v3
```

Can you update the changes to ☝️ so we follow the other skills and pin to the major version e.g. https://github.com/skills/continuous-integration/pull/19?